models/users.model.js

```diff
@@ -0,0 +1,18 @@
+const { normalizeQueryConfig } = require("pg/lib/utils");
+const db = require("../db/connection");
+const fs = require("fs/promises");
+
+const fetchUsers = () => {
+  return db
+    .query(
+      "SELECT * FROM users"
+    )
+    .then(({ rows }) => {
+      return rows;
+    })
+    .catch((err) => {
```

Comment:
lets remove this catch block here as we already have a catch in our controller so if this promise fails it will be caught in that catch block